requirements/base.in

```diff
@@ -2,7 +2,6 @@
 
 -c constraints.txt
 
-Django>=1.11,<2.3
```

Comment:
shouldn't we remove the constraint instead of removing the `Django` dependency itself